packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -743,7 +737,6 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 				this.mc.logger.sendPerformanceEvent({
 					eventName: "SummarizeAttemptDelay",
 					duration: retryAfterSeconds,
-					summaryNackDelay: ackNackResult.data?.retryAfterSeconds !== undefined,
```

Comment:
Can we infer this boolean from other stuff in the log?

Reply:
Yes, we know which stage failed

Reply:
Added it back.

Reply:
No one looks at this so it's not really interesting.